### ISSUE TITLE
install using docker -- basic instructions not working

### ISSUE THREAD

**Issue by shaunc** [directus/docs#336]
In https://docs.directus.io/installation/docker.html

Step 4 is:

> docker-compose run directus install --email email@example.com --password d1r3ctu5

This doesn't seem to work. From a shell inside directus container:

```
$ directus
bash: directus: command not found
```
In fact `directus` is available in `bin/directus`

Then the install process has been split into parts, it would seem.
```
root@directus-0:/var/directus# bin/directus help install

Directus Module Install Commands

	Configure Directus: 

		install:config -k my-project -h db_host -n db_name -u db_user -p db_pass -d directus_path -a super_admin_token


	Populate the Database Schema: 

		install:database -d directus_path


	Install Initial Configurations: 

		install:install -e admin_email -p admin_password -t site_name

```
I shouldn't have to do the first one, right, since I have directus configured to use environment config?
```
env | grep DIRECTUS
DIRECTUS_AUTH_PUBLICKEY=...
DIRECTUS_APP_ENV=production
DIRECTUS_DATABASE_PASSWORD=...
DIRECTUS_DATABASE_USERNAME=directus
DIRECTUS_DATABASE_HOST=directus-clouddoc-mysql
DIRECTUS_SERVICE_PORT_PORT_0=8085
DIRECTUS_DATABASE_NAME=clouddoc
DIRECTUS_PORT=tcp://172.20.40.41:8085
DIRECTUS_DATABASE_PORT=3306
DIRECTUS_PORT_8085_TCP=tcp://172.20.40.41:8085
DIRECTUS_INSTALL_EMAIL=...
DIRECTUS_INSTALL_PASSWORD=...
DIRECTUS_SERVICE_HOST=172.20.40.41
DIRECTUS_SERVICE_PORT=8085
DIRECTUS_PORT_8085_TCP_PORT=8085
DIRECTUS_PORT_8085_TCP_ADDR=172.20.40.41
DIRECTUS_PORT_8085_TCP_PROTO=tcp
DIRECTUS_USE_ENV=1
DIRECTUS_AUTH_SECRETKEY=...
```
I have done `bin/directus install:database` and `bin/directus install:install` (should be in documentation).
After forwarding container port 80 to 8080, I now see login, but when I login I get the error (after a long pause) 
![image](https://user-images.githubusercontent.com/193170/78693411-33bceb00-78c9-11ea-9136-ef543b250987.png)

Is this a port-forwarding issue? Or is it because the config wasn't properly read from the environment? Still debugging -- but help and documentation changes would be appreciated.

**Comment by WoLfulus** [comment]
it seems you're not running the command with docker-compose, but inside the container.

`directus` in the documentation refers to the service name within `docker-compose`, not the actual command that is being run inside the container.

invoking it through `docker-compose` will call the entrypoint script with the `install` argument, and will fall here

https://github.com/directus/docker/blob/master/images/apache/rootfs/var/directus/bin/directus-entrypoint#L89-L92

this is the actual script that installs it

https://github.com/directus/docker/blob/master/images/apache/rootfs/var/directus/bin/directus-entrypoint#L9-L64

if you don't want to use `docker-compose`, try running `directus-entrypoint install` instead

let me know if that works for you

**Comment by shaunc** [comment]
Ah ... thanks!

I'm actually using kubernetes with devspace, so no compose. The first thing I did was open a shell then do "install" (which of course didn't work) ... but I'd forgotten this could be a function in the entrypoint. So my next explanation was that there was just a typo in the instructions.

Sorry about that. (NB -- there does seem to be a helm chart, but it looks quite old and isn't mentioned in your documentation. I don't know if you would like a PR for it, or is it abandoned? Some comments in the documentation about kubernetes install would be helpful.... don't know if you want to have an open issue for that. Closing for now.)
